src/main/java/org/jenkinsci/plugins/saml/SamlEncryptionData.java

```diff
@@ -46,11 +47,11 @@ public String getKeystorePath() {
     }
 
     public String getKeystorePassword() {
```

Comment:
No, this must return `Secret`. Otherwise you leave a vulnerability: the secret will be round-tripped in plaintext, so anyone reconfiguring the form can just use View Source to see an existing password.

The Jenkins test harness will catch this mistake for you automatically *if* you have a functional test using, say, `JenkinsRule.configRoundtrip()` or otherwise using HtmlUnit to process the form.